Jetszxcki/LECCO-System
# feature: loans (incomplete)

## Description
- Just starting loans feature. 
- Added view and add for loans part (payrolls and payment_payrolls not yet implemented)

## To Test in your computer
- If you want to test in your computer. **NOTE: make sure local repo is not ahead or behind remote master branch**
- to copy remote branch to local repo
  - `git checkout --track origin/feature/loans`
  - you can revert back your local repo to remote/master branch after checking code using `git checkout --track origin/master`
  - You can fix any error you find in this branch if you want. *Just commit it to this branch tho*
    - Or tell jepp.
- If its running and no runtime errrors happening around. feel free to merge it (it can be automatically merged assuming master branch wasn't changed)

## Update
- Nag request, tas nag sarili pagmerge.
